styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: pink;
+  text-align: left;
+  font-family: "Lato", sans-serif;
+}
+h2 {
+  font-family: "Poppins", sans-serif;
+  line-height: 25px;
+}
+h3 {
+  font-size: 24px;
+  line-height: 18px;
+}
+
+h2,
+h3,
+p {
+  color: #5c585a;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 12px;
+}
+
+button {
+  background-color: white;
+  border: none;
+  bottom: 30px;
+  color: white;
+  cursor: pointer;
+  left: 140px;
+  margin: 5px;
+  padding: 12px 14px;
+  position: relative;
+}
```

Comment:
I did a modified version of your code in [this Glitch remix](https://glitch.com/edit/#!/remix-arielle-p2) that shows how to get this same layout without using the relative positioning.